### ISSUE TITLE
Wrong type declaration for Server in index.d.ts

### ISSUE THREAD

**Issue by Mikerosoft25** [Fabricio-191/valve-server-query#7]
Hello and first of all thank you for this package.

When trying to build my project I get the following error: 

```
error TS2694: Namespace '"@fabricio-191/valve-server-query". Server' has no exported member 'Data'.
280  export function Server(data?: Server.Data): Promise<Server>;
                                          ~~~~

Found 1 error in node_modules/@fabricio-191/valve-server-query/typings/index.d.ts:280
```

I am using your npm-package version 4.1.7 with typescript 4.9.4. 
I think the type declaration in `typings/index.d.ts` for the parameter of the Server function is wrong:
```
// Causing an error
export function Server(data?: Server.Data): Promise<Server>;
```

The `Data` Interface in line 207 of `typings/index.d.ts` is in the `MasterServer` namespace and not in the `Server` namespace.  But I think for this Server-Function the `Options` interface in the `Server` namespace should be used, shouldn't it? When changing the type declaration to `Server.Options` the build works:

```
// Works
export function Server(data?: Server.Options): Promise<Server>;
```

Could you please check if you have the same problem and update the `index.d.ts` with the correct type?

Thanks in advance!

**Comment by Fabricio-191** [comment]
Oops, solved in version 4.1.8  👍

**Comment by Mikerosoft25** [comment]
Thanks!
